src/components/Feed/Feed.tsx

```diff
@@ -50,14 +52,16 @@ const Feed = ({ userSentiment, articles, selectedArticles, findMatchingArticles,
      })
 
     return (
-      <div className="articles-container">
-      <div className="all-sections">
-        <SectionForm findMatchingArticles={ findMatchingArticles }/>
-      </div>
-        <section className="articles-display">
-          { foundArticleCards.length ? foundArticleCards : articleCards }
-        </section>
-      </div>
+      <main>
```

Comment:
Should this be a `section`? I was thinking Feed is only part of main... maybe our div surrounding App should be a `main` element .